jobs/aws/eks-distro/main-1-18-postsubmits.yaml

```diff
@@ -63,6 +67,10 @@ postsubmits:
             - -c
             - test $(($(date +%s) - 15)) -lt $(cat /status/pending)
           periodSeconds: 15
+        resources:
+          limits:
```

Comment:
Probably not a huge deal, but comparing the requests/limits on the presubmit job, this is a bit more resources than we give there.

https://github.com/aws/eks-distro-prow-jobs/blob/main/jobs/aws/eks-distro/kubernetes-1-18-presubmits.yaml#L77

Reply:
Yup that was on purpose as we have bigger nodes now